core/services/workflows/engine.go

```diff
@@ -88,7 +88,7 @@ func (e *Engine) resolveWorkflowCapabilities(ctx context.Context) error {
 	for _, t := range e.workflow.triggers {
 		tg, err := e.registry.GetTrigger(ctx, t.ID)
 		if err != nil {
-			e.logger.Errorf("failed to get trigger capability: %s", err)
+			e.logger.Errorf("capability id: %s failed to get trigger capability: %s", t.ID, err)
```

Comment:
For returned errors, i tried to keep the labelling consistent, but it's kind of awkward because we dont have structured errors like we do for logs.